docker-storage-setup.sh

```diff
@@ -54,12 +54,20 @@ set -e
 #
 # Note: lvm2 version should be same or higher than lvm2-2.02.112 for lvm
 #       thin pool functionality to work properly.
+
+# Check if the script is being run inside of a SPC container by examining $HOST
+# If set, PATH to use $HOST/usr/bin and $HOST/usr/sbin for host helper applications.
```

Comment:
I think for the time being we are just going to not run commands from the host.  The container needs to match the host as much as possible.  We need to see if this is a real problem.